### PR TITLE
feat(static): add human readable option for program headers

### PR DESCRIPTION
**Pull request by dacianpascu06**
## Description of change
Added KeyCommand HumanReadable that toggles the format for FileSiz and MemSiz in Tab::StaticAnalysis => Program Headers. Keybind is 's'

Closes Issue #47

## How has this been tested? (if applicable)
Verified the program to correctly toggle human readable on and off only on tab StaticAnalysis.

### CODE REVIEW

File: src/tui/state.rs [hunk 1]
```diff
@@ -353,6 +353,26 @@ impl<'a> State<'a> {
...
+                            .into_iter()
+                            .filter(|items| {
+                                self.input.value().is_empty()
+                                    || items.iter().any(|item| {
+                                        item.to_lowercase()
+                                            .contains(&self.input.value().to_lowercase())
+                                    })
+                            })
+                            .collect(),
+                    );
```
Comment (orhun) [inline src/tui/state.rs:373]: We can simply call:

```suggestion
                    self.handle_tab()?;
```

File: src/tui/state.rs [hunk 2]
```diff
@@ -451,6 +471,7 @@ impl<'a> State<'a> {
             Tab::StaticAnalysis => vec![
                 ("Enter", "Details"),
                 ("/", "Search"),
+                ("s", "Human Readable"),
```
Comment (orhun) [inline src/tui/state.rs:474]: Why do we use the `s` key here? We can go with `r` (`r`eadability) and change the description to "readability" or something similar. The latter is not necessary - my point is "s" key is not that intuitive :)

Also, I just realized pressing `r` on any tab re-runs the executable, we should probably fix that to happen on dynamic tab analysis only. We should probably fix that 🫠

File: src/elf/header.rs [hunk 3]
```diff
@@ -111,33 +113,64 @@ impl<'a> Property<'a> for FileHeaders {
 pub struct ProgramHeaders {
     /// Inner type.
     inner: Vec<ProgramHeader>,
+    /// Human readable format
+    human_readable: bool,
+}
+impl ProgramHeaders {
+    /// Toggle's the value for human_readable format
+    pub fn set_value(&mut self) {
```
Comment (orhun) [inline src/elf/header.rs:121]: `toggle_human_readable` would be a better name for this function I think.

**Review by orhun — CHANGES_REQUESTED** [review]
Thanks for the implementation! Tested and works - just needs some improvements 🐻

File: src/tui/state.rs (hunk 2 above)
Comment (dacianpascu06) [reply src/tui/state.rs:474]: r is already used in keycode::Char for TraceCalls.  It can't reach both patterns with the way the keycodes are designed.

**Comment by dacianpascu06** [comment]
@orhun (don't think force push sends notif)

**Review by orhun — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**